perses/app/relative_point_mutation_setup.py

```diff
@@ -21,10 +21,18 @@
 kT = kB * temperature
 beta = 1.0/kT
 ring_amino_acids = ['TYR', 'PHE', 'TRP', 'PRO', 'HIS']
+CL_CHARGE = unit.Quantity(value=-1.0, unit=unit.elementary_charge)
```

Comment:
Do these need to be hard-coded? If you have the `Topology` object and an unmodified system, can't you just pull these from the `System` object directly?

Reply:
I'm not sure we want to pull the charge from the system -- what if the system doesn't have any Na+ or Cl- ions (and instead has a different type of ion)? Making it system dependent might also propagate any errors in parametrization (even if the chance for this to occur is very small).

Reply:
I think the more important thing is that we want to use parameters that are compatible with the force field. This essentially hard-codes the force field, but when the water or ion model is updated, you'll be introducing weird mixtures of parameters that may not work together.

We can always fall back to defaults in there are no monovalent counterions in the system, but it is quite likely we will always want to be running with physiological or experimental salt concentration, rather than minimal neutralizing counterion concentrations.

Reply:
Fixed